tests/e2e/performance/test_pvc_clone_performance.py

```diff
@@ -76,96 +78,139 @@ def base_setup(self, interface_type, pvc_size, pvc_factory, pod_factory):
+
+        return int_number
+
     def test_clone_create_delete_performance(self, interface_type, pvc_size, file_size, teardown_factory):
         """
+        Write data (60% of PVC capacity) to the PVC created in setup
         Create single clone for an existing pvc,
-        Measure clone creation times
+        Measure clone creation time and speed
         Delete the created clone
-        Measure clone deletion time
-        Note: by increasing max_num_of_clones value you increase created clones number
+        Measure clone deletion time and speed
+        Note: by increasing max_num_of_clones value you increase number of the clones to be created/deleted
         """
-        logger.info(f"Running IO on pod {self.pod_obj.name}")
+
         file_name = self.pod_obj.name
-        logger.info(f"File {file_name} of {file_size} size was created during IO.")
-        self.pod_obj.run_io(
-            storage_type='fs', size=file_size, fio_filename=file_name, io_direction='wo', runtime=180, bs='1024K'
-        )
+        logger.info(f'Starting IO on the POD {self.pod_obj.name}')
+        # Going to run only write IO to fill the PVC for the before creating a clone
+        self.pod_obj.fillup_fs(size=file_size, fio_filename=file_name)
 
         # Wait for fio to finish
-        self.pod_obj.get_fio_results()
-        logger.info(f"Io completed on pod {self.pod_obj.name}.")
+        fio_result = self.pod_obj.get_fio_results()
+        err_count = fio_result.get('jobs')[0].get('error')
+        assert err_count == 0, (
+            f"IO error on pod {self.pod_obj.name}. "
+            f"FIO result: {fio_result}."
+        )
+        logger.info('IO on the PVC Finished')
 
         # Verify presence of the file on pvc
         file_path = pod.get_file_path(self.pod_obj, file_name)
         logger.info(f"Actual file path on the pod is {file_path}.")
         assert pod.check_file_existence(self.pod_obj, file_path), (
             f"File {file_name} does not exist"
         )
-        logger.info(f"File {file_name} exists in {self.pod_obj.name}")
+        logger.info(f"File {file_name} exists in {self.pod_obj.name}.")
 
         max_num_of_clones = 1
-        clone_creation_time_measures = []
+        clone_creation_measures = []
         clones_list = []
         timeout = 18000
         sc_name = self.pvc_obj.backed_sc
         parent_pvc = self.pvc_obj.name
         clone_yaml = constants.CSI_RBD_PVC_CLONE_YAML
         if interface_type == constants.CEPHFILESYSTEM:
             clone_yaml = constants.CSI_CEPHFS_PVC_CLONE_YAML
+        file_size_mb = self.convert_str_size_to_int(file_size)
 
         # creating single clone ( or many one by one if max_mum_of_clones > 1)
-        logger.info(f"Start creating {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size}GB.")
+        logger.info(f"Start creating {max_num_of_clones} clones on {interface_type} PVC of size {pvc_size} GB.")
```

Comment:
@ypersky1980 During below for loop execution would request to keep an eye on the memory growth of the "run-ci" command, we might observe https://github.com/red-hat-storage/ocs-ci/issues/2744 due to object creations in loop.

Reply:
@ramkiperiy I am worry, but I did not understand this comment. Can you please elaborate?

Reply:
@ramkiperiy talked to Avi regarding this. I think that I understand what you meant - to monitor process memory consumption during the test run, rught ?

Reply:
Yes correct, keep an eye on run-ci process memory consumption using top `top -n 1000 -b -E g | grep run-ci`